tools/phpunit/bootstrap.php

```diff
@@ -58,13 +58,15 @@
 	$plugin_name = 'performance-lab';
 }
 
+define( 'TESTS_PLUGIN_DIR', REPO_ROOT_DIR . "/plugins/$plugin_name/tests" );
```

Comment:
This constant should point to the root directory of the plugin, not the `tests` directory. That's the original purpose of the constant, and it IMO still makes sense as it simplifies to refer to anything within the plugin's directory, not just test-specific things. The only reason it's prefixed with `TESTS_` is that the constant is test specific.
```suggestion
define( 'TESTS_PLUGIN_DIR', REPO_ROOT_DIR . "/plugins/$plugin_name" );
```

Of course after this the references will need to be adjusted again. For many occurrences though, this effectively means leaving them like they used to be and making the PR diff much smaller. A huge part of the PR diff now is only because of this change to the `TESTS_PLUGIN_DIR` constant pointing to the plugin's `tests` dir.

Reply:
Done in 70de425.

Reply:
> The only reason it's prefixed with `TESTS_` is that the constant is test specific.

Ah, right. It's just the prefix. I was interpreting it as `PLUGIN_TESTS_DIR`. So if we wanted a `tests/` dir constant then we could have `TESTS_PLUGIN_TESTS_DIR` 😅  (Not needed.)